### DISCUSSION TITLE
osu!lazer mania should have the option to set whether your scroll speed is constant or scales with certain factors in maps

### DISCUSSION THREAD

**Discussion by khrocksgg** [ppy/osu#14979] [Ideas] [answered]
A lot of VSRG players end up with comfortable scroll speed ranges, which it's uncomfortable to go outside either end of. Normal osu!mania has the ability to have the scroll speed be constant between beatmaps, and it makes the experience a lot better, so it should be added to lazer as well, at least in my opinion.

**Comment by smoogipoo** [comment, marked as answer]
Have you played osu!lazer? This is already how it works and unlike stable there's no option to not have a constant scroll speed.

**Reply by khrocksgg** [reply]
as a matter of fact, i have played multiple maps where the scroll speed does change

they were converted from standard maps, but that shouldn't be a factor in this case

**Reply by peppy** [reply]
that's intended and also the case on stable.

**Reply by khrocksgg** [reply]
i know, but with stable i seem to remember an option to have the scroll speed always match what's set instead of scaling, which made mania a whole lot more playable

i'm suggesting that option be brought to lazer so that people who are comfortable with specific scroll speeds aren't forced to play outside their comfortable range to be able to play certain maps

**Reply by peppy** [reply]
There's no such option. Scroll speed adjustments set by mappers are intentional design choices by the mapper. These cannot be turned off.

Maybe we can add a mod which bypasses them, but it would come with a skill reduction weighting.

**Reply by bdach** [reply]
> Maybe we can add a mod which bypasses them

I believe this already exists in lazer, in fact (it's called "Constant Speed")

**Reply by peppy** [reply]
Amazing.

**Reply by khrocksgg** [reply]
> I believe this already exists in lazer, in fact (it's called "Constant Speed")

oh, it does? i genuinely didn't realise. whoops.
